setup.py

```diff
@@ -49,12 +49,21 @@
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```

Comment:
**Suggestion** 3.6 could be dropped. 3.7 is also at the end of it's life I wonder if that should be dropped too.

Reply:
Removed Python 3.6.
As for 3.7, at the time of this writing, is supported until the 27th of June, 2023. Some maintainers are strict about that, and so I will not be dropping it for this PR, but I think I'll drop it for the 5.0 upgrade.

Reply:
On the 5.0 PR, this commit should remove compatibility tests for Python 3.7:
https://github.com/springload/wagtail-django-recaptcha/pull/42/commits/06e7784708b10f4a29ee4e4cfd4b044e72e57857

Reply:
> Removed Python 3.6. As for 3.7, at the time of this writing, is supported until the 27th of June, 2023. Some maintainers are strict about that, and so I will not be dropping it for this PR, but I think I'll drop it for the 5.0 upgrade.

Thanks Kat I'm happy with that.